flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/WebSubmissionExtension.java

```diff
@@ -61,7 +61,7 @@ public WebSubmissionExtension(
 			Executor executor,
 			Time timeout) throws Exception {
 
-		webSubmissionHandlers = new ArrayList<>(5);
+		webSubmissionHandlers = new ArrayList<>(6);
```

Comment:
I think the following rule applies here. I would suggest to get rid of the capacity altogether.

> Set the initial capacity for a collection only if there is a good proven reason for that, otherwise do not clutter the code. In case of Maps it can be even deluding because the Map’s load factor effectively reduces the capacity.

https://flink.apache.org/contributing/code-style-and-quality-java.html#collections

Reply:
OK.